README.md

```diff
@@ -24,7 +24,7 @@ _(make sure you pin the latest stable release version after the `@`)_
 
 ```groovy title="nextflow.config"
 plugins {
-  id 'nf-validation@0.3.1'
+  id 'nf-validation'
```

Comment:
Any reason for this?

NB: The comment above doesn't really make sense any more if this is removed.

Reply:
Oh yeah I removed it to make it easier for us to maintain (since not specifying a version will by default use the latest one). I'll fix the comment above (or would you rather have it the way it was?)

Reply:
I feel like there was a reason that we didn't want to leave it unpinned, but now I can't remember what it was 🤔 

@mirpedrol can you remember?

Reply:
We leave it unpinned in the nf-core pipeline template to always install the latest version, so I think it makes sense to remove the version here too.
The downside is that it tries to pull the latest version for every run and it's a bit more difficult to run pipelines offline, but we can add a link to the [Nextflow documentation](https://github.com/nextflow-io/nextflow/blob/master/docs/plugins.md#offline-usage) (I think this version of the docs is not released yet, or I can't find it)

Reply:
Updated the comment to describe this behaviour.